fuzz/filter/positiveboundaryvalueanalysis.go

```diff
@@ -71,13 +70,27 @@ func (f *PositiveBoundaryValueAnalysisFilter) Apply(tok token.Token) (token.Toke
 
 		// middle
 		if l > 2 {
-			if err := tok.Permutation(uint(math.Ceil(float64(l) / 2.0))); err != nil {
-				panic(err)
+			if tok.From() < 0 && tok.To() > 0 {
+				// the boundaries are be -1, 0 and 1
```

Comment:
Since this changes the behaviour of the filter, please document the change above.